balena.yml

```diff
@@ -0,0 +1,31 @@
+name: "Nebra Helium Light"
+type: sw.application
+description: >-
+  Manage your Helium Light Gateway remotely using balenaCloud. 
+fleetcta: Internet all the things!
+post-provisioning: >-
+  ## Usage instructions
+  Once your device joins the fleet you'll need to allow some time for it to
+  download the application and start forwarding packets to the Helium Network.
+  For detailed instructions on how to use Nebra Helium Light check out
+  [our guides](https://helium.nebra.com).
+assets:
+  repository:
+    type: blob.asset
+    data:
+      url: "https://github.com/NebraLtd/light-hotspot-software"
+  logo:
+    type: blob.asset
+    data:
+      url: "https://cdn.shopify.com/s/files/1/0071/2281/3001/files/Nebra-Icon-Blue_bb758eaa-b10d-4ea6-a55e-babd69135b69.png?v=1620938774"
+  defaultDeviceType: raspberry-pi
+  supportedDeviceTypes:
+    - raspberry-pi
+    - raspberry-pi2
+    - raspberrypi3
+    - raspberrypi3-64
+    - raspberrypi4-64
+    - raspberrypi400-64
```

Comment:
I think I'm going to remove 400 as don't really think it's one to support.

Reply:
yeah probably not particularly relevant - removed